bin/jetify

```diff
@@ -9,12 +9,21 @@ rm -f $MAPPING_SED
 echo "Creating new sed command from $MAPPING_FILE"
 while IFS=, read -r from to
 do
-	echo "s/$from/$to/g;" >> $MAPPING_SED
+  echo "s/$from/$to/g;" >> $MAPPING_SED
 done <<< "$(cat $MAPPING_FILE)"
 echo "Command file for sed ready. Searching for files to jetify..."
 
-for FILE in `find $PROJECT_DIR \( -name "*.java" -o -name "*.xml" \) -type f -not -path '*/\.git*' | sort`; do
-  echo "jetifying file $FILE"
-  sed -i".bak" -f $MAPPING_SED $FILE
-  rm -f "$FILE.bak"
+jetify() {
+  echo "jetifying file $1"
+  sed -i -f $MAPPING_SED $FILE
+}
+
+N=4
+
+find . -type f -name "*_peaks.bed" ! -path "./tmp/*" ! -path "./scripts/*" 
```

Comment:
Could we maybe add ` ! -path "./examples/*" ` ?

Reply:
Hmm - that is potentially valid - an example should be a self-contained app, and to reach in to one and modify it unknowingly might break example assumptions. The example itself should depend on jetifier and run jetify via postinstall if it needs it. I would be okay with this provided the demonstrator continues to work: https://github.com/mikehardy/rn-androidx-demo

Reply:
adding that path didn't make it into the PR, if it's important to you please make a new PR and I'll add it in after testing